python/huggingfaceserver/huggingfaceserver/__main__.py

```diff
@@ -71,9 +71,10 @@ def list_of_strings(arg):
 )
 parser.add_argument(
     "--max_length",
+    dest="max_model_len",
```

Comment:
```suggestion
```

Reply:
This is to make sure we're writing to same argument and hence avoid doing `args.max_length or args.max_model_len`